modules/nlu/src/backend/application/index.ts

```diff
@@ -20,8 +20,11 @@ export class NLUApplication {
     this._queueTrainingOnBotMount = queueTrainingOnBotMount
   }
 
-  public async initialize() {
+  public async initialize(autoResumeQueue: boolean = false) {
```

Comment:
I don't like this boolean arg:
1. I don't like boolean args in general as in ends up looking like this: `app.initialize(true, true, false)` where you don't know what each parameters mean. I prefer `app.initialize({ resumeQueue: true })`
2. This boolean really just seem to be there for automated tests to keep passing because we don't use it in code and we don't plan on using it. Would prefer that in automated tests you switch all `app.initialize(true)` to :

```ts
app.initialize()
app.resumeTrainings() // for the test to pass
app.mountBot()
```